### PR TITLE
fix: allow removal of mailing address

### PR DESCRIPTION
**Pull request by ludtkemorgan**
* 2de28612c - fix: add error message for incorrect listing approval (#4134) 
* 0418793b6 - fix: allow removal of mailing address (#4130)

### CODE REVIEW

**Comment by ludtkemorgan** [comment]
This is blocked until this PR is merged as it uses the new toast setup: https://github.com/metrotranscom/doorway/pull/651

**Comment by ludtkemorgan** [comment]
This has been unblocked and can be reviewed

**Review by ColinBuyck — APPROVED** [review]
LGTM assuming that the failing tests are what @emilyjablonski mentioned in slack 🦋

**Pull request merged**
